ngrinder-core/src/main/java/net/grinder/util/NetworkUtils.java

```diff
@@ -223,7 +223,13 @@ public static List<Integer> getAvailablePorts(String ip, int size, int from, int
 		if (StringUtils.isNotBlank(ip)) {
 			try {
 				inetAddress = InetAddress.getByName(ip);
-			} catch (Exception e) {
+				if (!inetAddress.isReachable(MAX_REACHABLE_TIMEOUT)) {
+					processException(
+						"Can not check available ports because given local IP address '" + ip + "' is unreachable. " +
+							"Please check the `/etc/hosts` file or manually specify the local IP address in `${NGRINDER_HOME}/system.conf`."
+					);
+				}
```

Comment:
The issue #885 was hard to understand because the error message was not appropriate.
Adding reachability check and giving a detail error message will make it clearer.